src/index.html

```diff
@@ -7,8 +7,35 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo">
+      </div>
+      <nav class="links">
+        <ul class="headerList">
+          <li class="nav__li"><a href="#" class="is-active link">Apple</a></li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="is-active nav__link">
              Apple
            </a>
          </li>
          
```

it would be more readable and don't forget to add blank lines between child elements
also, check class names